### DISCUSSION TITLE
What is the servo motor equivalent of Webots?

### DISCUSSION THREAD

**Discussion by HimGautam** [cyberbotics/webots#4398] [Q&A]
In the small scale robotics projects we usually use servo motors which are controlled by pulse. I wanted to know what kind of motor in Webots can simulate a real world servo motor?

**Comment by omichel** [comment]
To implement a servo motor in Webots, you can combine a [RotationalMotor](https://cyberbotics.com/doc/reference/rotationalmotor) and a [PositionSensor](https://cyberbotics.com/doc/reference/positionsensor) in a [HingeJoint](https://cyberbotics.com/doc/reference/hingejoint). This is exemplified in the [Servo](https://github.com/cyberbotics/webots/blob/master/projects/devices/generic/protos/Servo.proto) which you can also use directly.

Also, in your case, it is possible to use a simple RotationalMotor and perform [control in position](https://cyberbotics.com/doc/reference/motor#position-control) to ensure the given position is reached by the motor.
